src/app/campaign-details/campaign-details.component.html

```diff
@@ -5,23 +5,28 @@ <h2>Banner Image here!</h2>
 <div class="details" *ngIf="campaign">
 	<div>
 		<h2>{{campaign.title}}</h2>
-    <h3 *ngIf="campaign.championName">Championed by {{campaign.championName}}</h3>
+    	<h3 *ngIf="campaign.championName">Championed by {{campaign.championName}}</h3>
```

Comment:
This indent looks weird - we should avoid literal tab characters as it makes this inconsistent across editors.

This seems to have changed for whitespace in general. Please make sure your editor is configured to respect the project's `.editorconfig` for HTML too @tbg-jasonfung